src/compiler/types.ts

```diff
@@ -4902,11 +4902,14 @@ namespace ts {
         // It is a good reminder of how much room we have left
         HasComputedFlags = 1 << 29, // Transform flags have been computed.
 
+        ContainsES2018 = 1 << 30,

```

Comment:
We will need to free up some space in transform flags as this is to large of a value. Refer to the comment above on line 4900.

Reply:
Maybe it should be done in another PR, before merging this one? Not sure what flags can be removed, though.

Reply:
Once #27410 lands, you should be able to choose a bit less than 29.